### PR TITLE
chore(git): correct codeowners file

### PR DESCRIPTION
**Pull request by sc979**
## Description

Fix codeowner team affectation errors and enhance team security scope

**Fixes** # (issue)
image

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+/.project       @centreon/centreon-build
+/Jenkinsfile    @centreon/centreon-build
+/selinux/       @centreon/centreon-build
+/project/       @centreon/centreon-build
+*.sh            @centreon/centreon-build
 
-.snyk				@centreon/centreon-security
-sonar-project.properties	@centreon/centreon-security
+/.snyk				            @centreon/centreon-security
+/sonar-project.properties	@centreon/centreon-security
```
Comment (sc979) [inline .github/CODEOWNERS:10]:
```suggestion
/.snyk				        @centreon/centreon-security
/sonar-project.properties	@centreon/centreon-security
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+/.git*          @centreon/centreon-build
+/.project       @centreon/centreon-build
+/Jenkinsfile    @centreon/centreon-build
+/selinux/       @centreon/centreon-build
+/project/       @centreon/centreon-build
+*.sh            @centreon/centreon-build
 
-.snyk				@centreon/centreon-security
-sonar-project.properties	@centreon/centreon-security
+/.snyk				        @centreon/centreon-security
```
Comment (sc979) [inline .github/CODEOWNERS:9]:
```suggestion
/.snyk				    @centreon/centreon-security
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+/project/       @centreon/centreon-build
+*.sh            @centreon/centreon-build
 
-.snyk				@centreon/centreon-security
-sonar-project.properties	@centreon/centreon-security
+/.snyk				@centreon/centreon-security
+/sonar-project.properties	@centreon/centreon-security
 
-*.po            @ghost
+# @ghost seems deprecated
```
Comment (kduret) [inline .github/CODEOWNERS:12]: Ghost is not deprecated, it's just an unknown user

**Review by kduret — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (sc979) [reply .github/CODEOWNERS:12]: Documentation team creation is an alternative to consider ;)

**Pull request merged**